File: Course_01/C++ Language/Vetores/soma_vetores.cpp
#include <iostream>

using namespace std;

int main()
{
    int n, i;

    cout << "Quantos valores vai ter cada vetor? ";
    cin >> n;

    double a[n], b[n];

    cout << "Digite os valores do vetor A:" << endl;

    for (i = 0; i < n; i++) {
        cin >> a[i];
    }

    cout << "Digite os valores do vetor B:" << endl;

    for (i = 0; i < n; i++) {
        cin >> b[i];
    }

    cout << "VETOR RESULTANTE:" << endl;

    for (i = 0; i < n; i++) {
        cout << a[i] + b[i] << endl;
    }
    return 0;
}

File: Course_01/Python/Vetores/soma_vetores.py
n = int(input("Quantos valores vai ter cada vetor? "))

vetA = [0 for x in range(n)]
vetB = [0 for x in range(n)]
vetSoma = [0 for x in range(n)]

print("Digite os valores do vetor A:")
for i in range(0, n):
    vetA[i] = int(input())

print("Digite os valores do vetor B:")
for i in range(0, n):
    vetB[i] = int(input())

for i in range(0, n):
    vetSoma[i] = vetA[i] + vetB[i]

print("Vetor resultante:")
for i in range(0, n):
    print(vetSoma[i])
